db/src/commonMain/kotlin/com/islamversity/db/model/Calligraphy.kt

```diff
@@ -0,0 +1,89 @@
+package com.islamversity.db.model
+
+import com.squareup.sqldelight.ColumnAdapter
+
+inline class LanguageCode(val lang: String) {
```

Comment:
It seems the inline feature must be added to the [gradle](https://github.com/islamversity/Reyan/blob/seyed/add-database-schema-and-datasources/db/build.gradle.kts) file.